src/main/java/seedu/application/model/application/Application.java

```diff
@@ -171,6 +174,12 @@ public boolean isSameApplication(Application otherApplication) {
                 && otherApplication.position.equals(position);
     }
 
+    private void applicationDateIsInFuture(Date date) {
+        if (date.value.isAfter(LocalDate.now())) {
```

Comment:
```suggestion
        if (date.isAfter(LocalDate.now())) {
```
can define a `isAfter()` method in `Date` class to adhere to SLAP